endpoints/events/accounts_test.go

```diff
@@ -0,0 +1,141 @@
+package events
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/stored_requests"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+// Mock AccountsFetcher
+type mockAccountsFetcher struct {
+	Fail          bool
+	Error         error
+	EventsEnabled bool
+}
+
+func (e *mockAccountsFetcher) FetchAccount(ctx context.Context, accountID string) (json.RawMessage, []error) {
+	if e.Fail {
+		return nil, []error{e.Error}
+	}
+
+	var acc = &config.Account{
+		ID:            accountID,
+		EventsEnabled: false,
+	}
+
+	if e.EventsEnabled {
+		acc.EventsEnabled = true
+	}
+
+	s, _ := json.Marshal(acc)
```

Comment:
Please never ignore an error response. I suggest accepting `t *testing.T` as an argument, add a call to `t.Helper()` to tell Go this is a util method, then do a `t.Fatal` if there's a marshal error.